modules/core/src/main/scala/org/tessellation/http/routes/DebugTrustRoutes.scala

```diff
@@ -0,0 +1,34 @@
+package org.tessellation.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.domain.trust.storage.TrustStorage
+import org.tessellation.infrastructure.trust.storage.TrustStorage.TrustStore
+
+import org.http4s._
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class DebugTrustRoutes[F[_]: Async](
+  trustStorage: TrustStorage[F]
+) extends Http4sDsl[F] {
+  private[routes] val prefixPath = "/debug/trust"
```

Comment:
I think it should be part of `DebugRoutes`

Reply:
The `DebugRoutes` and the storages and services used by it are in the `sdk` module. `TrustStorage` is in the `core` module. I would have to move all the domain parts of `TrustStorage` and its service to `sdk` to add it to the `DebugRoutes`. This includes `snapshotInterval` and `delayToUpdateCurrent`.

Reply:
The domain parts of `TrustStorage` have been moved to the `sdk` module.  `DebugTrustRoutes` remain in the `core` module.